tf_files-1.0/aws/modules/eks/cloud.tf

```diff
@@ -43,6 +43,7 @@ module "jupyter_pool" {
 
 module "workflow_pool" {
   count                         = var.deploy_workflow ? 1 : 0
+  scale_in_protection           = true
```

Comment:
Can we use the variable `scale_in_protection` here too and set it to be true by default?